src/modules/dataFetching/components/GenericTableWithData.tsx

```diff
@@ -108,7 +108,7 @@ function allFieldColumns<T>(recordType: string): ColumnDef<T>[] {
 }
 
 function allFieldFilters<T>(
-  recordType: string,
+  recordType: string, // < not needed
```

Comment:
Made this change in another PR here: https://github.com/greenriver/hmis-frontend/pull/749